api/package.json

```diff
@@ -85,6 +85,7 @@
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.2",
     "rxjs": "^6.6.3",
+    "sparkpost": "^2.1.4",
```

Comment:
Should we pin version?

Reply:
we're not pinning other versions, we use lock, so not sure about this

Reply:
the new ones are usually pinned and we I believe [we should follow this convention](https://docs.renovatebot.com/dependency-pinning/).